src/layers/core/arc-layer/arc-layer.js

```diff
@@ -82,7 +84,8 @@ export default class ArcLayer extends BaseLayer {
 
     return new Model({
       program: new Program(gl, {
-        vs: assembleShader(gl, {vs: glslify('./arc-layer-vertex.glsl')}),
+        vs: assembleShader(gl, {vs: glslify('./arc-layer-vertex.glsl')},
+          getPlatformShaderDefines(gl)),
```

Comment:
The idea here was that `assembleShader` would automatically add the platform defines to the output. I.e. it should not be necessary to call `getPlatformShaderDefines` when using `assembleShader`